x/blog/codec.proto

```diff
@@ -50,17 +50,9 @@ message Article {
   // DeleteAt defines deletion time of the article.
   // Could be nil if there is not a time of deletion
   int64 delete_at = 10 [(gogoproto.casttype) = "github.com/iov-one/weave.UnixTime"];
-}
-
-// DeleteArticleTask is used for representing scheduled task pk. Used when deleting the task
-message DeleteArticleTask {
-  weave.Metadata metadata = 1;
-  // PrimaryKey is the unique identifier of the task
-  bytes pk = 2 [(gogoproto.customname) = "PrimaryKey"];
-  // ArticleKey is the pk of the article that will be deleted
-  bytes article_key = 3 [(gogoproto.customname) = "ArticleKey"];
-  // TaskOwner is the creator of the task
-  bytes task_owner = 4 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
+  // Delete task ID holds the ID of the asynchronous task that is scheduled to
+  // delete the task id at delete_at
```

Comment:
This is a very weird sentence. How about two changes:

1. Rename `task_id` to `delete_task_id`
2. Change the comment to
    > Delete Task ID holds an ID of a tasks scheduled to delete this article. This value can be empty if no deletion task was scheduled.